nixos/modules/services/misc/spice-autorandr.nix

```diff
@@ -0,0 +1,26 @@
+{ config, pkgs, lib, ... }:
+
+let
+  cfg = config.services.spice-autorandr;
+in
+{
+  options = {
+    services.spice-autorandr = {
+      enable = lib.mkEnableOption (lib.mdDoc "Enable spice-autorandr service that will automatically resize display to match SPICE client window size.");
```

Comment:
This "Enable" prefix is redundant because `mkEnableOption` will already include that for you.